src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -8,16 +8,16 @@ namespace Microsoft.CodeAnalysis.CSharp.CodeStyle
     internal static class CSharpCodeStyleOptions
     {
         // TODO: get sign off on public api changes.
-        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CSharpCodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
+        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
```

Comment:
feature name will be used to find right serializer I believe. it might have changed. but otherwise, I think you need to change serializer as well?